src/api/provisionFormOptionsData.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+	provisionFormOptions: 
+	[
+		{
+			services: [
+				{
+					identifier: 'github'
+				}
+			],
+			automationSoftwares: [
+				'ansible'
+			],
+			serverProviders: [
+				'digitalOcean'
```

Comment:
Which convention should we use for names like this? lower case, camel case or something_like this?